buildSrc/src/main/groovy/org/grails/Page.groovy

```diff
@@ -10,7 +10,7 @@ class Page implements Content {
 
     @Override
     String getPath() {
-        filename
+        filename?.replaceAll('\\\\', '/')
```

Comment:
[`RenderSiteTask.highlightMenu`](https://github.com/grails/grails-static-website/blob/830b143fe883e59da66eda1250983f1d41abc32b/buildSrc/src/main/groovy/org/grails/gradle/RenderSiteTask.groovy#L218) takes a `String path` parameter and supplies it to `replaceAll`. When the user is building on Windows, the path format is as follows:
```
C:\\example\\path\\to\\file
```
Because of the backslashes, `replaceAll` treats this string as a regex escape character. This causes a build failure due to `Illegal/unsupported escape sequence`. 

Replacing `\\` with `/` allows us to still reference the file but avoid confusing the regex parser.